jupyter_book/page.py

```diff
@@ -86,3 +92,12 @@ def build_page(path_ntbk, path_html_output, path_media_output=None, execute=Fals
     writer.write(html, resources, notebook_name=notebook_name)
     if verbose:
         print("Finished writing notebook to {}".format(path_html_output))
+
+
+def _is_jupytext_file(ntbk):
+    """Infer whether a notebook node was created from a Jupytext Markdown file.
+
+    Right now, this just tries to guess based on whether there's a particular piece of
+    metadata in the notebook. Not sure if this is actually correct though...
+    """
```

Comment:
I think you want to test that the metadata `notebook_metadata_filter` is `-all`. I will explain this with a bit more detail at https://github.com/mwouts/jupytext/issues/320